.vscode/settings.json

```diff
@@ -10,5 +10,16 @@
     "strings": true
   },
   "tailwindCSS.classFunctions": ["tw", "tw`", "twClassName", "twMerge"],
-  "tailwindCSS.validate": true
+  "tailwindCSS.validate": true,
+  "editor.formatOnSave": true,
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "[javascript]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+  "[typescript]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+  "[typescriptreact]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  }
```

Comment:
Enables automatic code formatting on file save using Prettier as the default formatter. This ensures consistent code style across the project without requiring manual formatting. If for some reason Format on Save is not working we can run `yarn prettier . --write`


https://github.com/user-attachments/assets/7c1b4ab7-1f43-45ef-9411-f66c030e6092